Todor_Garnev/lesson10/hm1/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Задача</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
+</head>
+
+<body>
+    <section class="container-sprite">
+        <header>
+            <h1>Social navigation</h1>
+            <h3>With image sprite</h3>
+        </header>
+
+        <main>
+            <ul>
+                <li><a href="#"><span class="facebook"></span>facebook</a></li>
+                <li><a href="#"><span class="google"></span>Google +</a></li>
+                <li><a href="#"><span class="instagram"></span>instagram</a></li>
+                <li><a href="#"><span class="pinterest"></span>pinterest</a></li>
+                <li><a href="#"><span class="twitter"></span>twitter</a></li>
+            </ul>
+        </main>
+    </section>
+
+    <section class="container-fa">
+        <header>
+            <h1>Social navigation</h1>
+            <h3>With font awesome</h3>
+        </header>
+
+        <main>
+            <ul>
+                <li><a href="#">facebook<span class="fa fa-facebook round"></span></a></li>
```

Comment:
what if i told you font-awesome has circles too ? :)

http://fontawesome.io/examples/#stacked